src/main/java/com/agilethought/atsceapi/service/HistoryServiceImpl.java

```diff
@@ -37,4 +41,10 @@ public NewHistoryResponse createHistory(NewHistoryRequest request) {
         History savedHistory = historyRepository.save(newHistory);
         return new NewHistoryResponse(savedHistory.getId());
     }
+    
+	@Override
+	public List<History> getAllHistory(String id) {
```

Comment:
if I sent `/api/v1/histories?userId=` , what is the response?

Reply:
An empty array